deep-learning/_posts/2017-05-17-tiramisu.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: review
+title:  "The One Hundred Layers Tiramisu: Fully Convolutional DenseNets for Semantic Segmentation"
+tags:   deep-learning CNN segmentation
+author: Clément Zotti
+pdf:   "https://arxiv.org/pdf/1611.09326.pdf"
+
+cite:
+  authors: "S.Jégou, M.Drozdzal, D.Vasquez, A.Romero, Y.Bengio"
+  title:   "The One Hundred Layers Tiramisu: Fully Convolutional DenseNets for Semantic Segmentation"
+  venue:   "arXiv 2016"
+---
+
+This papers implement all the well known advanced in the image segmentation community and make it work together.
+
+List of the methods used:
+
+* Dense blocks
+* U-Net architecture
+* Using IOU loss
+
+
+## Dense blocks
+
+The dense block provide a low increase of parameters when keeping a good feature extraction.
+
+<div align="middle">
+     <img src="/deep-learning/images/tiramisu/dense_blocks.png">
+</div>
+
+## U-Net
+
+The U-Net is well known to have a good performance on various images segmentation dataset.
+
+## Architecture
+
+The network architecture is a straight U-Net with more stage of comvolution in each steps.
```

Comment:
A stage it's the number of `layers` inside a dense block, but it's not regular convolutional layers it's the `Layer` show in the figure in the `Layers` part of the post. I can change the name which one did you prefer ? They call it `n_layers_per_block` in their code.